docs/src/basicproperties.md

```diff
@@ -8,7 +8,9 @@ The following is an overview of functions for accessing graph properties. For fu
 - `vertices`: Iterable object of all graph vertices.
 - `edges`: Iterable object of all graph edges.
 - `has_vertex`: Checks for whether graph includes a vertex.
-- `has_edge`: Checks for whether graph includes an edge.
+- `has_edge(g, s, d)`: Checks for whether graph includes an edge from a given source `s` to a given destination `d`.
+-  `has_edge(g, e)` will return true if there is an edge in g that satisfies `e == f` for any `f ∈ edges(g)`. This is a strict equality test that may not limit itself for testing that just there exists any edge between the source and destination.
```

Comment:
I think I understand what you're trying to say here, but it's a bit unclear. Perhaps
> This is a strict equality test that, depending on the implementation of the edge type, may be broader than checking that an edge exists between `src(e)` and `dst(e)`.

Reply:
sorry just copied your previous text

Reply:
Yeah. The fact that I didn't recognize the text means that it's bad :) Sorry about that.